### PR TITLE
Reflex Item - React

### PR DESCRIPTION
**Pull request by spar36**
Reflex Item - Reactive tier 1

Issue #236

### CODE REVIEW

File: game/scripts/npc/items/reflex_cores/react/item_react.txt [hunk 1]
```diff
@@ -1,66 +1,93 @@
...
+		"ItemBaseLevel"						"1"
+
+
+		// Item Info
+		//-------------------------------------------------------------------------------------------------------------
+		"AbilityManaCost"					"100"
+		"ItemCost"								"5000"
+		"ItemShopTags"						"agi;str;int;attack_speed;move_speed;hard_to_tag"
+		"ItemQuality"							"epic"
+		"ItemAliases"							"wing chun"
```
Comment (chrisinajar) [inline game/scripts/npc/items/reflex_cores/react/item_react.txt]: ? 😂

**Comment by chrisinajar** [comment]
it looks like you have a duplicate item_react and item_reactive. also the tier 2 and 3 need their a/b/c indications, since there are 2 tier 2 and 3 tier 3 versions.

**Comment by chrisinajar** [comment]
actually it doesn't look like 2 or 3 do anything, they shouldn't be modified.

**Comment by warpdragon** [comment]
Go true. If theres a duplicate get rid of it b4 PR itll make my life easier later

**Comment by spar36** [comment]
I only edited reactive tier 1. I didnt edit the other files at all, only moved them into the reflex_items/react folder.

**Pull request merged**